apps/api/init_comps_v2.sh

```diff
@@ -0,0 +1,485 @@
+#!/bin/bash
```

Comment:
why is this script being added as part of this PR?  Seems unrelated?

Reply:
It was an accident, will remove it.